app/models/in_person_enrollment.rb

```diff
@@ -60,17 +60,25 @@ def self.needs_late_email_reminder(early_benchmark, late_benchmark)
   def self.needs_usps_status_check(check_interval)
     where(status: :pending).
       and(
-        where(status_check_attempted_at: check_interval).
-        or(where(status_check_attempted_at: nil)),
+        where(last_batch_claim_at: check_interval).
+        or(where(last_batch_claim_at: nil)),
+      ).
+      order(status_check_attempted_at: :asc)
```

Comment:
The changes in usage here mean that we don't rely on a specific ordering of the enrollments. We can probably remove the `order` part from this function.

Reply:
We would still want to hit the 'oldest' first of those that might be getting a second try. This way, if the job runs for a long time, it will at least try the older ones first.

Reply:
I think that may help a bit, but not as much as with the previous strategy since we will be re-selecting the records with the same ordering later.

Reply:
Whoops, you're right, I was thinking of the next instance. Thanks!